### ISSUE TITLE
No proper list conversion

### ISSUE THREAD

**Issue by jimmyMaci** [anubhavshukla/properties-to-yaml-converter#1]
Hello @anubhavshukla,

i want to convert a properties list that looks as follows:
```
application.public-paths=/v1/jwt/authenticate,/v1/jwt/ispublic,/v1/auth/signin
```
with the following source code:
```
import com.anubhavshukla.p2y.converter.PropertiesToYamlConverter;
...
PropertiesToYamlConverter propertiesToYamlConverter = new PropertiesToYamlConverter();
String result = propertiesToYamlConverter.toYamlString(propertiesFilename);
```
the actual output is:
```
application:
  public-paths: /v1/jwt/authenticate,/v1/jwt/ispublic,/v1/auth/signin
```
but should be:
```
app:
  public-paths:
      - /v1/jwt/authenticate
      - /v1/jwt/ispublic
      - /v1/auth/signin
```
Do i something wrong?

Cheers
@jimmyMaci

**Comment by anubhavshukla** [comment]
@jimmyMaci ,

Thanks for pointing this out. The list functionality is currently not supported. I'll release an update ASAP.

**Comment by anubhavshukla** [comment]
Fix available in latest release 1.0.2
